view-creator-framework/src/test/resources/sample-view-generation-spec.conf

```diff
@@ -19,12 +19,13 @@ pinot = {
   timeColumn = creation_time_millis
   timeUnit = MILLISECONDS
   dimensionColumns = [name, creation_time_millis, id_sha, friends, properties__KEYS, properties__VALUES]
+  dateTimeColumns = [start_time_millis, end_time_millis]
```

Comment:
What would be the impact of querying on this column? I think, we discussed but forgot.

Reply:
Got it, it will behave the same as our existing time field.
```
"timeFieldSpec": {
    "incomingGranularitySpec": {
      "name": "start_time_millis",
      "dataType": "LONG",
      "timeType": "MILLISECONDS"
    },
    "defaultNullValue": 0
  }
```

Reply:
Yes. That's correct